src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -103,12 +105,23 @@ export class UserListComponent implements OnInit {
         prop: 'enabled',
         flexGrow: 1,
         cellTransformation: CellTemplate.checkIcon
+      },
+      {
+        name: this.i18n('Password expiration date'),
+        prop: 'pwdExpirationDate',
+        flexGrow: 1,
+        pipe: this.cdDatePipe
```

Comment:
nit:  (maybe for a future PR) what about using the same color scheme (yellow / red) for dates that are about to expire or are already expired in the users list table?

Reply:
https://tracker.ceph.com/issues/43058